### DISCUSSION TITLE
Always netstandard20 with Unity?

### DISCUSSION THREAD

**Discussion by Jenix-Park** [melanchall/drywetmidi#198] [Q&A] [answered]
On [this page](https://melanchall.github.io/drywetmidi/articles/dev/Using-in-Unity.html), you said, "for master branch take DryWetMIDI.<release_number>-bin-netstandard20.zip archive..".
Is still the netstandard20 version recommended over net45 even with the latest Unity Editor?

**Comment by melanchall** [comment, marked as answer]
For latest Unity you should definitely use .NET Standard binary. Please note that there is [DryWetMIDI asset](https://assetstore.unity.com/packages/tools/audio/drywetmidi-222171) on the Unity Asset Store to get rid of manual steps.

**Reply by Jenix-Park** [reply]
What happens if I use the net45 version?

Older Unity Editor definitely needed the API Compatibility level of .NET Standard 2.0, but starting the version 2018.1, the .NET 4.x has been supported as stated [here](https://docs.microsoft.com/en-us/visualstudio/gamedev/unity/unity-scripting-upgrade).
So I put the DryWetMIDI-net45 files in some of my projects manually in the past and had no issues at all so far.

Do I need to switch all of them?
I don't use MIDI devices and playback features in those projects, by the way.

**Reply by melanchall** [reply]
Nothing bad will happen with .NET Framework version of the library. But .NET Standard is more modern technology than .NET Framework, so please use it. I believe .NET Framework support was added to cover more developers and to provide ability to use more old 3d party libraries written on .NET Framework.

More than that, from the linked article:

> Unity recommends .NET Standard 2.0 for new projects.

So I recommend you to use .NET Standard.
